tests/conftest.py

```diff
@@ -182,15 +225,16 @@ def whitelistTradingContracts(self):
             self.controller.addToWhitelist(self.contracts[name].address)
 
     def initializeAllContracts(self):
-        for contractName in CONTRACTS:
+        contractsToInitialize = ['mutex','Cash','orders','completeSets','makeOrder','takeBidOrder','takeAskOrder','takeOrder','cancelOrder','trade','claimProceeds','tradingEscapeHatch','ordersFetcher']
```

Comment:
Yeah, that is the _exact_ argument for the move.  :)  I'm generally a fan of defining things scoped as tightly as possible.  In this case, since they are used in only one place they could be inlined (which I actually started with) but I found that having the variable name improved readability by associating a name with the collection, so I hoisted it up one scope level.